src/en/authors-charm-actions.md

```diff
@@ -0,0 +1,317 @@
+# Actions for the Charm author
+
+---
+
+ - ### [Defining Actions](#defining-actions)
```

Comment:
I like this table of contents, but this might actually instead become something we generated automatically from the markdown structure (in fact I think I'll start on that next week)